File: Prolog/fizzbuzz.pl
/*
Author-Aastha Singh
FizzBuzz in PROLOG

*/
fizz3(X) :- 0 is mod(X, 3).
buzz5(X) :- 0 is mod(X, 5).

FizzBuzz(X) :- 
    (fizz3(X),buzz5(X))
    	-> write('FizzBuzz') 
    	; fizz3(X) 
    		-> write('Fizz') 
    		; buzz5(X) 
    			-> write('Buzz') 
    			; write(X).
loop(100).
loop(n) :-n>0, FizzBuzz(101-n), nl. 
S is n-1, loop(S).

File: Delphi/fizzBuzz.pas
// simple fizzbuzz in delphi
// Author: @augustoscher
// Hactoberfest 2021

procedure FizzBuzz;
var
  i:Integer;
begin
  for i := 1 to 100 do
  begin
    case i mod 15 of
      3, 6, 9, 12:
        WriteLn('Fizz');
      5, 10:
        WriteLn('Buzz');
      0:
        WriteLn('FizzBuzz');
      else
        WriteLn(IntToStr(i));
    end;
  end;
end;
